packages/block-library/src/button/edit.js

```diff
@@ -224,6 +224,7 @@ function ButtonEdit( props ) {
 					} }
 					anchorRef={ ref?.current }
 					focusOnMount={ isEditingURL ? 'firstElement' : false }
+					__unstableSlotName={ 'block-toolbar' }
```

Comment:
>(Just for the context) This bug appeared after https://github.com/WordPress/gutenberg/pull/34956.

Based on the shared PR, it seems to me that setting the slot name to `__unstable-block-tools-after` would be better. -cc @ellatrix 

>These popovers need to use a slot in the block tools component instead, so it's the next tabbable element after the content.

Reply:
@ntsekouras Thanks for the suggestion, slot name has been updated.